faunadb-scala/src/main/scala/faunadb/types/RequestParameters.scala

```diff
@@ -0,0 +1,29 @@
+package faunadb.types
```

Comment:
i noticed there was nothing in this package yet.

What's the point of this?

Can we just use the commons class?